### DISCUSSION TITLE
Extracting LORA

### DISCUSSION THREAD

**Discussion by Barry-Bot1** [TheLastBen/fast-stable-diffusion#1968] [General]
Is is possible to Extract a models LORA in this UI ?

**Comment by TheLastBen** [comment]
it would take more than 25GB of RAM which colab doesn't offer for free

**Comment by Omenizer** [comment]
It used to be possible with SuperMerger Extension and paid colab, I extracted a few of my dreambooths. But the Lora extraction in Supermerger is broken now and my precious compute units are rotting...

**Reply by TheLastBen** [reply]
I will add that feature to the dreambooth colab soon

**Reply by ClaraSanders** [reply]
I also use Supermerger but I can't merge LORAS to a checkpoint. I get a strange error like that I'm storing data in two devices instead of one.
RuntimeError: Expected all tensors to be on the same device, but found at least two devices, cuda:0 and cpu!
Any idea about how to solve it?

**Reply by Omenizer** [reply]
> I will add that feature to the dreambooth colab soon

Oooh that's awesome!

**Reply by ClaraSanders** [reply]
> It used to be possible with SuperMerger Extension and paid colab, I extracted a few of my dreambooths. But the Lora extraction in Supermerger is broken now and my precious compute units are rotting...

Hey, Omenizer can you tell me how you extracted your Dreambooths? Were you able to merge Loras to checkpoints? I also have Google Colab Pro but I only can merge checkpoints and Loras. I get the error mentioned above: RuntimeError: Expected all tensors to be on the same device, but found at least two devices, cuda:0 and cpu!

**Reply by meunumerotim333** [reply]
> I will add that feature to the dreambooth colab soon

Ben, can you create a Colab Notebook to train Lora's? Similar as Dreambooth Colab. The ones that I've found is so complicated. Yours is so simple.

**Reply by Omenizer** [reply]
That would be nice as well!

**Reply by TheLastBen** [reply]
> Ben, can you create a Colab Notebook to train Lora's? Similar as Dreambooth Colab. The ones that I've found is so complicated. Yours is so simple.

Hopefully soon
